app/src/main/java/com/example/myfridgehome/dto/Recipe.kt

```diff
@@ -2,6 +2,6 @@ package com.example.myfridgehome.dto
 
 data class Recipe(var title : String = "") {
     override fun toString(): String {
-        return title
+        return "$title"
```

Comment:
Made this change in case there is requirement of another variable and won't create any confusion later.

Reply:
These two lines are doing the same thing with the same output, though the one added in code review requires more typing.  I'd keep it simple, and only add the "$" reference if/when this function needs to return more than just the variable value returned.